src/extdev/development/record-validation.adoc

```diff
@@ -0,0 +1,113 @@
+include::../../variables.adoc[]
+
+= Record Validation
+
+Automation scripts provide the ability to validate records before they are created or updated.
+This helps us define a system that is more resilient to the dangers of malformed user input.
+
+[IMPORTANT]
+====
+Record validation from a script can be performed by scripts that are executed on:
```

Comment:
Can we mention all `after...` event counterparts and why are not usable in this context?

Reply:
Opened a new issue; will add asap